prime-angular-frontend/src/app/shared/components/enrollee/enrollee-property/enrollee-property.component.html

```diff
@@ -4,14 +4,18 @@
 
     <span>{{ title }}</span>
 
-    <button *ngIf="hasError"
+    <button *ngIf="hasError && !showError"
             mat-icon-button
             type="button"
             [disabled]="true">
       <mat-icon [matTooltip]="errorMessage"
                 matTooltipPosition="after"
                 color="warn">error</mat-icon>
     </button>
+    <span class="error-text d-flex" *ngIf="hasError && showError">
```

Comment:
Minor thing, can you make the `*ngIf` come before `class`

Reply:
updated as suggested.